frontend/src/static/css/v_zoom.scss

```diff
@@ -1,5 +1,13 @@
 @import 'colors';
 
+.toolbar {
```

Comment:
We have CSS for this in `panels.css` maybe can utilize that? Otherwise, it's just duplicate.

Reply:
The CSS here forces the links to appear on two lines. I think its use is different from panels.css?